### ISSUE TITLE
Breakage: Google Forms

### ISSUE THREAD

**Issue by outloudvi** [kkapsner/CanvasBlocker#378]
## Description
Some parts of Google Forms are breaking with CanvasBlocker, examples:
* [Example 1: Unable to submit some forms with reCAPTCHA](https://docs.google.com/forms/d/e/1FAIpQLScGKF5OHiY-Zg1THpTUqgpuoeJPBELRK6RDFX84_pKLsAhwtw/viewform)

  (This form is created dedicated for testing, so feel free to test on this.)

* [Example 2: Cannot see the list of users' forms](https://docs.google.com/forms/u/0/) (Need google account logined)

Console log of example 1:
```
TypeError: documentWriteDescriptor is undefined
iframeProtection.js:146:25
Content Security Policy: Ignoring “'unsafe-inline'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “https:” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “http:” within script-src: ‘strict-dynamic’ specified
Content Security Policy: The report URI (about:blank) should be an HTTP or HTTPS URI.
Content Security Policy: The page’s settings observed the loading of a resource at https://www.google.com/recaptcha/api2/webworker.js?hl=en&v=v1561357937155 (“worker-src”). A CSP report is being sent.
Content Security Policy: Ignoring “'unsafe-inline'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “https:” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “http:” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “'unsafe-inline'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “https:” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “http:” within script-src: ‘strict-dynamic’ specified
```

Console log of example 2:

```
Error: Permission denied to access property "message"
2 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:152
TypeError: "documentWriteDescriptor is undefined" 0:61:325
Error: Permission denied to access property "message"
2 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:152
uncaught exception: CustomError: Error in protected function: Permission denied to access property "message" 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:328
uncaught exception: CustomError: Error in protected function: Module errback failures: CustomError: Error in protected function: Deferred has already fired 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:328
uncaught exception: CustomError: Error in protected function: Permission denied to access property "then" 4 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:328
uncaught exception: CustomError: Error in protected function: Permission denied to access property "message" 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:328
Error: Permission denied to access property "message"
3813710213-homescreen_binary_i18n_core__zh_cn.js:294:152
uncaught exception: CustomError: Error in protected function: Permission denied to access property "then" 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:328
Error: Permission denied to access property "message"
4 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:152
Empty string passed to getElementById(). 2 3813710213-homescreen_binary_i18n_core__zh_cn.js:523:143
Error: Permission denied to access property "message"
8 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:152
Error: Permission denied to access property "message"
6 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:152
TypeError: documentWriteDescriptor is undefined
iframeProtection.js:146:25
Error: Permission denied to access property "message"
2 3813710213-homescreen_binary_i18n_core__zh_cn.js:294:152
Empty string passed to getElementById().
```

## Expected Behavior
These sites are working as expected.

## Current Behavior
These sites are broken.

## Context
After disabling CanvasBlocker, these sites are working again.

## Your Environment
* CanvasBlocker Version used: 0.5.9.1
* Firefox version incl. 32- or 64-bit: 64 bit
* Operating System and version (desktop or mobile): desktop, Arch Linux
* Installed addons: uBlock Origin, uMatrix, Privacy Badger and many extensions that would be not related to this.

## Your Settings
``` json
{
	"logLevel": 1,
	"urlSettings": [],
	"hiddenSettings": {},
	"expandStatus": {},
	"displayHiddenSettings": false,
	"whiteList": "",
	"sessionWhiteList": "",
	"blackList": "",
	"blockMode": "fake",
	"protectedCanvasPart": "readout",
	"minFakeSize": 1,
	"maxFakeSize": 0,
	"rng": "nonPersistent",
	"protectedAPIFeatures": {},
	"useCanvasCache": true,
	"ignoreFrequentColors": 0,
	"minColors": 0,
	"fakeAlphaChannel": false,
	"persistentRndStorage": "",
	"persistentIncognitoRndStorage": "",
	"storePersistentRnd": false,
	"persistentRndClearIntervalValue": 0,
	"persistentRndClearIntervalUnit": "days",
	"lastPersistentRndClearing": 1562135238440,
	"sharePersistentRndBetweenDomains": false,
	"askOnlyOnce": "individual",
	"askDenyMode": "block",
	"showCanvasWhileAsking": true,
	"showNotifications": true,
	"highlightPageAction": "none",
	"highlightBrowserAction": "color",
	"displayBadge": true,
	"storeNotificationData": false,
	"storeImageForInspection": false,
	"ignoreList": "",
	"ignoredAPIs": {},
	"showCallingFile": false,
	"showCompleteCallingStack": false,
	"enableStackList": false,
	"stackList": "",
	"protectAudio": true,
	"audioFakeRate": "100",
	"audioNoiseLevel": "minimal",
	"useAudioCache": true,
	"audioUseFixedIndices": true,
	"audioFixedIndices": "25",
	"historyLengthThreshold": 2,
	"protectWindow": false,
	"protectDOMRect": true,
	"domRectIntegerFactor": 4,
	"blockDataURLs": true,
	"protectNavigator": false,
	"navigatorDetails": {},
	"displayAdvancedSettings": true,
	"displayDescriptions": false,
	"theme": "auto",
	"dontShowOptionsOnUpdate": false,
	"isStillDefault": false,
	"storageVersion": 0.5
}
```

**Comment by outloudvi** [comment]
Seems to be fixed by 0.5.11.

**Comment by kkapsner** [comment]
Nevertheless thanks for the input and the testing in 0.5.11
